app/src/pages/loginPage/loginPage.tsx

```diff
@@ -39,9 +76,9 @@ export const LoginPage = (props: ILoginPageProps): React.ReactElement => {
             <Stack isFullHeight={true} isFullWidth={true} shouldAddGutters={true} direction={Direction.Vertical} childAlignment={Alignment.Center}>
               <Text variant='header3'>Welcome Back :)</Text>
               <Spacing variant={PaddingSize.Wide} />
-              <SingleLineInput placeholderText='Email Address' inputType={InputType.Email} value={email} onValueChanged={setEmail}/>
+              <SingleLineInput inputWrapperVariant={emailError ? 'error' : 'success'} messageText={emailError || undefined} placeholderText='Email Address' inputType={InputType.Email} value={email} onValueChanged={onEmailTyped}/>
               <Spacing variant={PaddingSize.Wide} />
-              <SingleLineInput placeholderText='Password' inputType={InputType.Password} value={password} onValueChanged={setPassword}/>
+              <SingleLineInput inputWrapperVariant={passwordError ? 'error' : 'success'} messageText={passwordError || undefined} placeholderText='Password' inputType={InputType.Password} value={password} onValueChanged={onPasswordTyped}/>
```

Comment:
would be good to split these jsx over multiple lines once they have more than one prop. Also I think the `inputWrapperVariant` might need a case where there are no errors or succes? or is the message never defined in that case?

Reply:
i just changed it to be undefined in the case of lack of error. success doesn't really feel right here